mmseg/models/backbones/resnest.py

```diff
@@ -0,0 +1,310 @@
+import math
+
+import torch
+import torch.nn as nn
+import torch.nn.functional as F
+import torch.utils.checkpoint as cp
+from mmcv.cnn import build_conv_layer, build_norm_layer
+
+from ..builder import BACKBONES
+from ..utils import ResLayer
+from .resnet import Bottleneck as _Bottleneck
+from .resnet import ResNetV1d
+
+
+class RSoftmax(nn.Module):
+    """Radix Softmax module in ``SplAtConv2d``.
+
+    Args:
+        radix (int): Radix of input. groups (int): Groups of input.
+    """
+
+    def __init__(self, radix, groups):
+        super().__init__()
+        self.radix = radix
+        self.groups = groups
+
+    def forward(self, x):
+        batch = x.size(0)
+        if self.radix > 1:
+            x = x.view(batch, self.groups, self.radix, -1).transpose(1, 2)
```

Comment:
Is the `transpose` necessary? Can we specify `dim=2` for softmax?

Reply:
`x` is reshaped to original shape afterward, which means the stack order of `self.group` and `self.radix` is changed (although `self.group=1` in current config).